### DISCUSSION TITLE
Coupon/Promotions value

### DISCUSSION THREAD

**Discussion by Gerrysde** [sergioteula/python-amazon-paapi#80] [Help] [answered]
I tried this fantastic wrapper and I love it, but I was wondering where I can find the discount percentage of a product that has a coupon. I tought it was the "promotions" object but it's always null. I'm Italy, maybe this can help

**Comment by sergioteula** [comment, marked as answer]
Hello, thanks for your nice words. Regarding your doubt, I'm sorry to tell you that coupons and promotions such as flash and day prices are not included in the API, so there's no way to get them using this module.

**Reply by Gerrysde** [reply]
Very sad, but thank you for your answer.
